string_calculator.rb

```diff
@@ -0,0 +1,42 @@
+class StringCalculator
+  def add(string_of_numbers = "")
+    sum = 0
+    separate_string = string_of_numbers.split(",")
```

Comment:
I would postpone splitting the input string by comma until I've detected what the delimiter actually is. It may not be comma.